eip155/caip10.md

```diff
@@ -21,13 +21,33 @@ While Ethereum "accounts" were the unstated norm in the definition of
 [CAIP-10][], there are still some particularities of the syntax that should be
 specified for the unfamiliar.  
 
+### Special case of EOA
+
+To express Ethereum account without reference to any specific [EIP-155][] chain ID, CAIP-10 identifiers can use the special `chainId` segment `0`.
+
+In the Ethereum account system, "externally owned accounts" (i.e. "offchain" wallets) are controlled by user agents and not by on-chain entities (i.e. deployed or deployable smart contract code). 
+The special chainId `0` SHOULD only be used for accounts that can be used in off-chain contexts, i.e. without the use of an Ethereum node.
+
+Note that a given address cannot be assumed to work on all current and future networks with an [EIP-155][] `chainId`, which may express or derive addresses differently for a given private key; a user account expressed with `chainId` segment of 0 is assumed to control the same account on `chainId` 1 and most ethereum chains, but MAY control a different account or additional accounts on networks that extend the account or addressing model of Ethereum, assume different key or hash derivations, etc.
+
 ## Syntax
 
 Ethereum addresses were, historically, case-insensitive and normalized to use
 all-lowercase letters (`abcdef`) like most hexadecimal numeric types.  With the
-ratification of [EIP55][], however, a particular normalization of lowercase- and
+ratification of [EIP-55][], however, a particular normalization of lowercase- and
 uppercase- `abcdefABCDEF` characters was invented as an efficient form of
-checksum. See [EIP55][] for specification.
+checksum. See [EIP-55][] for specification.
+
+The chain ID will be used to represent blockchain except special case of 0 as chainID to represent EOA.
+
+The `chain_id` string will be ammended as follows:
+
+```
+chain_id:    namespace + ":" + network_id + ":" + reference
+network_id:    [0-9]{1,19}
+namespace:   [-a-z0-9]{3,8}
+reference:   0x[-_a-fA-F0-9]{1,32}
```

Comment:
```suggestion
reference:   0x[a-fA-F0-9]{1,32}
```
Whoops, missed this the first time.  Eth addresses can't have `-`s or `_`s in them! the CAIP-10 regex is much looser because it has to accomodate all kinds of addressing notations, but I think we can limit it to conventional EVM addresses here, at least until any L2s with custom addresses start complaining?